paimon-core/src/main/java/org/apache/paimon/io/KeyValueDataFileWriter.java

```diff
@@ -195,7 +195,8 @@ public DataFileMeta result() throws IOException {
                 deleteRecordCount,
                 indexResult.embeddedIndexBytes(),
                 fileSource,
-                valueStatsPair.getKey());
+                valueStatsPair.getKey(),
+                null);
```

Comment:
When externalPath is not null in DataFileMeta?
The config of externalPath will add in other pr?

Reply:
Yes